packages/context/src/fetch.ts

```diff
@@ -25,9 +18,26 @@ export default async function fetch(
   // the init opts to add the proxy agent
   if (proxy) {
     init = Object.assign({}, init, {
-      agent: HttpsProxyAgent(proxy),
+      agent: new HttpsProxyAgent(proxy),
```

Comment:
There was a good reason to use the `HttpsProxyAgent` without `new`. Did you verify this? I think it was related to the `https-proxy-agent` exposing this class in a wrong way. Maybe this has been fixed in the meantime...